optashift-employee-rostering-shared/src/main/java/org/optaplanner/openshift/employeerostering/shared/common/TimeSlotUtils.java

```diff
@@ -7,6 +7,10 @@
 
 public class TimeSlotUtils {
 
+    public static LocalDate toLocalDate(OffsetDateTime time) {
+        return LocalDate.of(time.getYear(), time.getMonthValue(), time.getDayOfMonth());
+    }
+
     public static LocalTime toLocalTime(OffsetDateTime time) {
```

Comment:
delete this one too (see above)

Reply:
See above (we should probably update our emulated library, it is very out of date)